packages/libs/react-ui/src/components/Notification/Notification.stories.tsx

```diff
@@ -27,7 +28,7 @@ const meta: Meta<
     color: {
       options: Object.keys(
         cardColorVariants,
-      ) as (keyof typeof cardColorVariants)[],
+      ).filter(e => e !== 'tertiary' && e !== 'secondary') as (NotificationColors)[],
```

Comment:
I think you can simplify this and just use Object.keys(color) to do this. I believe it will also already be typed for you

Reply:
Yeah my implementation of this was a bit rushed in between other things since Isa asked. My bad, I have refactored it to a different approach.